src/main.c

```diff
@@ -19,5 +19,48 @@
 
 int main(int argc, char **argv)
 {
-    return SuricataMain(argc, argv);
```

Comment:
I like how this style breaks things down a little bit more, helps one understand what happens when, improves readability a bit.